include/ed/entity.h

```diff
@@ -52,6 +54,10 @@ class Entity
     inline geo::ShapeConstPtr shape() const { return shape_; }
     void setShape(const geo::ShapeConstPtr& shape);
 
+    inline std::map<std::string, geo::ShapeConstPtr> volumes() const { return volumes_; }
```

Comment:
Given that we also get `addVolume` and `removeVolume`, I guess this should return a `const` map, probably as reference? (unless multiple threads change this? not entirely sure)

Reply:
There will never be two threads writing to the same entity instance. It is better if it is a const map